_i18n/en.yml

```diff
@@ -559,6 +559,20 @@ mining:
   software_para: Note that some miners may have developer fees.
   support: Support
   supportp: If you have questions or just want to confront with fellow miners, come chat on Monero Pools. On <a href="#monero-pools:monero.social">Matrix</a> and <a href="irc://irc.libera.chat/#monero-pools">Libera</a>.
+  p2poolh: "P2Pool: The best of both solo and pool mining"
+  p2poolnew: >
+    P2Pool is a new clever way of mining Monero, which allows to receive frequent payouts like if you are mining in a pool, but while mining solo. It's a Peer-To-Peer mining pool that gives you full control over your Monero node and what it mines. More details in <a href="{{ site.baseurl }}/2021/10/05/p2pool-released.html">the announcement post.
```

Comment:
P2Pool is a clever new way of mining Monero, which allows miners to receive the frequent payouts offered by pools without needing to trust a centralized pool. P2Pool is a Peer-To-Peer mining pool that gives miners full control over their Monero node and what it mines. More details in <a href="{{ site.baseurl }}/2021/10/05/p2pool-released.html">the announcement post.